Source/ORTS.Common/SystemInfo.cs

```diff
@@ -67,7 +67,9 @@ static SystemInfo()
             }
             catch (ManagementException error)
             {
-                Trace.WriteLine(error);
+                // Likely to catch multiple exceptions like:
+                // Exception thrown: 'System.IO.InvalidDataException' in ORTS.Menu.dll
+                //Trace.WriteLine(error);
```

Comment:
Did we have to remove the trace? These errors are useful diagnostics in the log file

Reply:
Restored